tests/system/large/test_location.py

```diff
@@ -66,7 +66,9 @@ def test_bq_location_default():
     _assert_bq_execution_location(session)
 
 
-@pytest.mark.parametrize("bigquery_location", config.ALL_BIGQUERY_LOCATIONS)
+@pytest.mark.parametrize(
+    "bigquery_location", sorted(bigframes.constants.ALL_BIGQUERY_LOCATIONS)
```

Comment:
Optional: add a comment like "sort to avoid nondeterminism?"

Reply:
```suggestion
    "bigquery_location",
    # Sort the set to avoid nondeterminism.
    sorted(bigframes.constants.ALL_BIGQUERY_LOCATIONS),
```